files/default/test-neutron-ha-tool.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+import collections
+ha_tool = __import__("neutron-ha-tool")
```

Comment:
problem is that the filename contains dashes. I looked for other solutions, and will switch to using the `importlib` module, that might read a bit better. The good solution would be to rename the file so it does not contain dashes.